modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -1741,7 +1741,14 @@ label:enterprise-edition[Enterprise Edition]
 [frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
 |===
 |Description
-a|Databases may be created from an existing 'seed' (a database backup or dump) stored at some source URI. Different types of seed source are supported by different implementations of `com.neo4j.dbms.seeding.SeedProvider`. For example, seeds stored at `s3://` and `https://` URIs are supported by the builtin `S3SeedProvider` and `URLConnectionSeedProvider` respectively. This list specifies enabled seed providers. If a seed source (URI scheme) is supported by multiple providers in the list, the first matching provider will be used. If the list is set to empty, the seed from uri functionality is effectively disabled.
+a|Databases may be created from an existing _seed_ (a database backup or dump) stored at some source URI.
+Different types of seed source are supported by different implementations of `com.neo4j.dbms.seeding.SeedProvider`.
+There are two available values, the built-in `S3SeedProvider` and `URLConnectionSeedProvider`.
+Seeds stored at `S3` are supported by `S3SeedProvider` and seeds stored at `file`, `ftp`,`http`, `https`, and URIs are supported by `URLConnectionSeedProvider`.
```

Comment:
I would phrase this slightly differently:
Seeds addressed with "s3:" URIs are supported by `S3SeedProvider` and seeds addressed by `file`, `ftp`,`http`, `https` URIs are supported by `URLConnectionSeedProvider`.